frontend/src/tests/components/commons/SideNavigation.test.js

```diff
@@ -11,10 +11,12 @@ const TEXT2 = "test2";
 const TEXT3 = "test3";
 const TEXT4 = "test4";
 
+window.HTMLElement.prototype.scrollIntoView = function() {}; // dummy for the scrollIntoView call within SideNavigation's changeNode function
```

Comment:
This is called stubbing a function and totally makes sense to do here! Can we move the comment to the line before so that it's easier to read, and call it a stub instead of a dummy function?

Reply:
Fixed in e49057c